text_formatter.go

```diff
@@ -78,6 +82,22 @@ func (f *TextFormatter) init(entry *Entry) {
 	}
 }
 
+func (f *TextFormatter) isColored() bool {
+	isColored := f.ForceColors || f.isTerminal
+
+	if f.OverrideColors {
```

Comment:
the new `OverrideColor` will enable the `CLICOLOR*` env overrides, this will not change the default behavior